lib/redis-backend.js

```diff
@@ -55,10 +56,32 @@ RedisBackend.prototype = {
 	      .end();
 
     key = this.bucketKey(bucket, key);
-    
+
     this.redis.smembers(key, cb);
   },
-  
+
+  /**
+    Gets an array of the contents of the specified keys.
+  */
+  getAll : function(bucket, keys, cb){
+    contract(arguments)
+      .params('string', 'array', 'function')
+      .end();
+
+    var redisKeys = this.bucketKey(bucket, keys);
+    var batch = this.redis.batch();
+    var self = this;
+
+    redisKeys.map(function(key) {
+      batch.smembers(key, self.redis.print);
+    });
+
```

Comment:
I am using this approach in another project: https://github.com/OptimalBits/bull/blob/master/lib/scripts.js